src/components/editMeasure/testCases/components/statusHandler/StatusHandlerMessage.tsx

```diff
@@ -86,18 +86,25 @@ export function createImportMessage(
   successfulImportsWithWarnings: TestCaseImportOutcome[],
   testDataId: string
 ) {
+  let failedMessage = `(${successfulImports}) test case(s) were imported. The following (
+              ${failedImports.length}) test case(s) could not be imported. Please
+              ensure that your formatting is correct and try again.`;
+  if (failedImports?.length > 0) {
+    const lockedTestCases = failedImports.filter((fi) =>
+      fi.message?.includes("The test case is locked by another user")
+    );
+    if (lockedTestCases?.length > 0) {
+      failedMessage = `Test case import was successful, but the following test cases could not be imported because they are in-use by another user and cannot be overwritten:`;
```

Comment:
what happens if I've some other issue along with lock issue? would this override the message at line 89?